// if tabbed is launched with the -g flag,
// tabbed becomes fixed and it sized cannot be changed
// this is EXTREMELY annoying if you want to do this...
//
// see my "x9term" script in my bin repo for an example
//
--- tabbed/tabbed.c	Mon Feb 24 23:07:54 2020
+++ tabbed/tabbed.c	Mon Feb 24 23:08:26 2020
@@ -921,7 +921,7 @@
 
 void
 setup(void) {
-	int bitm, tx, ty, tw, th, dh, dw, isfixed;
+	int bitm, tx, ty, tw, th, dh, dw;
 	XClassHint class_hint;
 	XSizeHints *size_hint;
 
@@ -948,7 +948,6 @@
 	wy = 0;
 	ww = 800;
 	wh = 600;
-	isfixed = 0;
 
 	if(geometry) {
 		tx = ty = tw = th = 0;
@@ -966,8 +965,6 @@
 			wx = -1;
 		if(bitm & YNegative && wy == 0)
 			wy = -1;
-		if(bitm & (HeightValue|WidthValue))
-			isfixed = 1;
 
 		dw = DisplayWidth(dpy, screen);
 		dh = DisplayHeight(dpy, screen);
@@ -1000,15 +997,9 @@
 	XSetClassHint(dpy, win, &class_hint);
 
 	size_hint = XAllocSizeHints();
-	if(!isfixed) {
-		size_hint->flags = PSize;
-		size_hint->height = wh;
-		size_hint->width = ww;
-	} else {
-		size_hint->flags = PMaxSize | PMinSize;
-		size_hint->min_width = size_hint->max_width = ww;
-		size_hint->min_height = size_hint->max_height = wh;
-	}
+    size_hint->flags = PSize;
+    size_hint->height = wh;
+    size_hint->width = ww;
 	XSetWMProperties(dpy, win, NULL, NULL, NULL, 0, size_hint, NULL, NULL);
 	XFree(size_hint);
 
